docs/man/docker-run.1.md

```diff
@@ -336,6 +336,10 @@ mount is not running.
 default, the volumes are mounted in the same mode (read write or read only) as
 the reference container.
 
+   If the location of the volume from the specified container overlaps with
```

Comment:
@moxiegirl ok - picked up your edits, with just minor tweaks. See how it looks now...